.ruby-version

```diff
@@ -0,0 +1 @@
+2.1.2
```

Comment:
The helps ruby version managers, like [rbenv](https://github.com/rbenv/rbenv), to use the correct version of ruby